polly/include/polly/ScopInfo.h

```diff
@@ -1494,7 +1494,7 @@ class ScopStmt final {
   /// @param Access  The access to add.
   /// @param Prepend If true, will add @p Access before all other instructions
   ///                (instead of appending it).
-  void addAccess(MemoryAccess *Access, bool Preprend = false);
+  void addAccess(MemoryAccess *Access, bool Prepend = false);
```

Comment:
Code change: Extra review, please.

This change aligns with the spelling at:
https://github.com/llvm/llvm-project/blob/d7e561b913d2a75c7c1807bf1c1e0bddc270a2b3/polly/lib/Analysis/ScopInfo.cpp#L1127

Reply:
LGTM